Schemas/botproject-schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$id": "https://schemas.botframework.com/schemas/botprojects/v0.1/worskspace.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$version": "0.1",
+  "title": "Virtual Assistant and Skills workspace schema",
+  "description": "This is a schema that captures the relationship between the virtual assistant and its skills.",
+  "type": "object",
+  "required": ["$schema", "name", "workspace", "skills"],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "format": "uri",
+      "description": "The schema to verify this workspace file against"
+    },
+    "name": {
+      "type": "string",
+      "description": "Name of the Virtual Assistant or Root Bot."
+    },
+    "workspace": {
+      "type": "string",
+      "description": "Path to the virtual assistant or Root Bot. Relative to the workspace file or an absolute path."
+    },
+    "provisioningProfile": {
+      "type": "string",
+      "description": "Path to provisioning profile."
+    },
+    "skills": {
```

Comment:
I think we should update the 'skills' property to be an object to match how the skills are configured in the settings. It also ensures each skill has a unique name.

```
{
  "skills": {
    "type": "object",
    "additionalProperties": {
      "$ref": "#/definitions/skill"
    }
  }
}
```